.github/workflows/wp-plugin-check-action.yml

```diff
@@ -0,0 +1,48 @@
+name: 'WordPress.org plugin directory guidelines'
+on: # rebuild any PRs and main branch changes
+  workflow_call:
+  pull_request:
+  push:
+    branches:
+    - main
+
+jobs:
+  wp-org-plugin-guidelines:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: latest
+          coverage: none
+          tools: wp-cli
+
+      - name: Install latest version of dist-archive-command
+        run: wp package install wp-cli/dist-archive-command:dev-main
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          # Make sure to use the same version like WordPress/gutenberg.
+          node-version-file: '.nvmrc'
+          cache: npm
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build plugin
+        run: |
+          npm run build
+          wp dist-archive . ./gatherpress.zip
+          mkdir tmp-build
+          unzip gatherpress.zip -d tmp-build
```

Comment:
@carstingaxion the error looks like it's related to the plugin not being activated. on activation it creates the tables if they do not exist. maybe try `wp plugin activate gatherpress` here?

Reply:
You may be right to some point, but I think at this time, _we_ don't even have a WordPress running, where we could activate anything. This happens in the next lines. I [asked for help and suggested your idea](https://github.com/swissspidy/wp-plugin-check-action/issues/61); let's see and wait a little.

Reply:
You don't need to activate anything.

Right now I don't see why this _warning_ with the database table happens, as I can't reproduce it locally. But you should be able to just ignore it, it's just a notice/warning.

The GitHub Action is still working fine.

The only _error_ is that you are using an `Update URI` header, which is disallowed on dotorg :-)

But it looks like you removed that already in 9a18e9f656cefbec3e475fa94eb0b2a74b2c86b4

Reply:
Thanks again for helping  @swissspidy!

With [1.0.4](https://github.com/swissspidy/wp-plugin-check-action/releases) in place, I'm happy to run this again.